cmd/infra/powervs/service_id.go

```diff
@@ -193,6 +203,10 @@ func extractServiceIDFromCRN(crn string) string {
 	return crnL[len(crnL)-1]
 }
 
+func updateCRYaml(crYaml, serviceInstanceValue string) string {
+	return fmt.Sprintf(crYaml, serviceInstanceValue)
```

Comment:
Its good practice to use templating here.

Reply:
Added the provision to use templating,
please let me know, if any further changes are required?